test/goss/local/health.yml

```diff
@@ -0,0 +1,32 @@
+user:
+  aeternity:
+    exists: true
+  master:
+    exists: true
+
+service:
+  sshd:
+    enabled: true
+    running: true
+
+port:
+  tcp:22:
+    listening: true
+    ip:
+    - 0.0.0.0
+
+port:
+  tcp:3015:
```

Comment:
i think these three `tcp:` should be under single `port:` because that way only the last will be parsed